rllib/tests/test_client.py

```diff
@@ -0,0 +1,92 @@
+import os
```

Comment:
Could we call this test_ray_client.py instead (same for the test case above)? To distinguish between an RLlib PolicyClient and the ray client?

Reply:
Updated!